src/Groups/perm.jl

```diff
@@ -220,6 +266,25 @@ function cperm(L::AbstractVector{T}...) where T <: IntegerUnion
    end
 end
 
+@doc Markdown.doc"""
+    cperm(L::Vector{Vector{T}}) where T <: IntegerUnion
+
+cperm can also handle cycles passed in inside of a vector
```

Comment:
This implicitly refers to another doc string that the user cannot see. Don't do that.
Make it self-contained, or merge the two docstrings.

Separately: I think this also needs a variant that takes a permgroup `g` as first argument, just like the other `cperm` method

Reply:
All docs are merged, and there is now a new variant